help/de/Help_ch_GettingHelp.xml

```diff
@@ -0,0 +1,173 @@
+
+      <important>
+        <para>Es wird dringend empfohlen, dieses Handbuch zu lesen, wenn Sie mit der Nutzung von 
+          &app; neu beginnen oder mit den Konzepten der Buchhaltung nicht vertraut sind.</para>
+      </important>
+    </sect2>
+
+    <sect2 id="wiki">
+      <title>&app; Wiki</title>
+      <!-- Translators: if wiki pages in your language exist, add or replace the links in this section
+          see also https://wiki.gnucash.org/wiki/Special:Categories for your language or region code -->
+      <para>Eine große Anzahl an weniger formeller, dafür aber <emphasis>aktuellerer</emphasis> Dokumentation, sowohl über
+        &app; selbst als auch über dessen Wartung und Entwicklung, finden Sie in dem 
+        <ulink url="&url-wiki-de;">&app; Wiki</ulink>; dort sollte die Seite
+        <ulink url="&url-wiki-faq-de;">Häufig gestellten Fragen (FAQ)</ulink>
+        die erste Anlaufstelle sein, wenn Sie Schwierigkeiten bei der Verwendung von &app; haben.</para>
+      <tip>
+        <para>Sehr hilfreich ist auch die  
+          <ulink url="&url-wiki-search;">Suchseite</ulink>.</para>
+      </tip>
+    </sect2>
+
+    <sect2 id="more_help">
+      <title>&app; Online Unterstützung</title>
+      <!-- Translators: if a mailing list in your language exists, add or replace the links in this section
+          see https://lists.gnucash.org/mailman/listinfo/ -->
+
+      <bridgehead>Mailing Liste</bridgehead>
+      <para>Die weitere wichtige Quelle für die Unterstützung der Anwender ist die 
+        deutschsprachige <ulink url="mailto:gnucash-de@gnucash.org">User Mailing List</ulink>. 
+        Falls Sie eine Forum-ähnliche Web-Darstellung bevorzugen, können Sie diese über 
+        <ulink url="&url-mail-nabble;">Nabble</ulink> erreichen. Sie müssen sich
+        <ulink url="&url-mail-li;gnucash-user">registrieren</ulink>
+        bevor Sie einen Beitrag schreiben, auch wenn Sie Nabble benutzen.</para>
+
+      <bridgehead>IRC</bridgehead>
+      <para>Einige der Entwickler und erfahrene Benutzer verfolgen den
+        <ulink url="&url-irc;">#gnucash-Kanal bei irc.gnome.org</ulink>.
+        Sie sind meist auch mit etwas anderem beschäftigt und natürlich nicht immer an ihrem
+        Computer. Loggen Sie sich ein, stellen Sie Ihre Frage, und bleiben Sie eingeloggt; es kann 
+        einige Stunden dauern, bis Ihre Frage bemerkt und beantwortet wird.
+        Um zu sehen, ob Sie etwas verpasst haben 
+        <ulink url="&url-logs-irc;">prüfen Sie die IRC-Protokolle</ulink>.
+        Die Anforderungen, Netiquette und andere Details werden in der
+        <ulink url="&url-wiki-irc;">IRC wiki Seite</ulink> erläutert.</para>
+    </sect2>
+
+    <sect2 id="website">
+      <title>&app; Website</title>
+
+      <para>Die <ulink url="&url-www;">&app; Homepage</ulink>
+        hat weitere Details zu diesen Kanälen. Dort finden Sie auch Verweise
+        auf weitere nützliche Informationen wie z.B. das Bug-Tracking-System.</para>
```

Comment:
Mit dem Absatz bin ich übrigens in der Englischen Version überhaupt nicht glücklich.
Das "dynamische Element dort sind die News (Release, Wartung).
Der eher statische Teil enthält viele Links zu Downloads, etc,
Ein Vorteil gegenüber dem Wiki sind die vielen Übersetzungen.

Falls du da noch eine Idee hast, wie man das besser formulieren kann …

Reply:
Vielleicht so:
Auf der <ulink url="&url-www;">&app; Homepage</ulink> werden in vielen Sprachen Informationen zu den neusten Release-Version, deren Verbesserungen und Neuerungen sowie Wartungszyklen gegeben. Die Möglichkeiten zum Download für geläufige Betriebssysteme, auch als &flatpak;, sowie Hinweise zur Installation werden dort ebenso bereitgestellt wie auch weiterführende Links zum Wiki.

Reply:
Ich denke, ich habe jetzt alles wesentliche drin: commit 2651ea2

Jetzt mach noch mal ein Reset auf commit b253f96 und versuch das mal mit dem rebase (auf .commit 2651ea2),
Anschließend kannst du 
1. deinen commit ammenden und 
2. deinen Zweig force pushen.

Reply:
Scheint ja prima geklappt zu haben, aber schau dir nochmal das englische diff an. Ich habe da noch ein paar Kleinigkeiten ergänzt wie Comunity beim Wiki.

Reply:
aarrgh, und ich hatte gedacht alles gefunden zu haben. Ist jetzt korrigiert.